src/gl/color_mode.js

```diff
@@ -20,19 +20,15 @@ class ColorMode {
 
     static Replace: BlendFuncType;
 
-    static disabled(): ColorMode {
-        return new ColorMode(ColorMode.Replace, Color.transparent, [false, false, false, false]);
-    }
-
-    static unblended(): ColorMode {
-        return new ColorMode(ColorMode.Replace, Color.transparent, [true, true, true, true]);
-    }
-
-    static alphaBlended(): ColorMode {
-        return new ColorMode([ONE, ONE_MINUS_SRC_ALPHA], Color.transparent, [true, true, true, true]);
-    }
+    static disabled: ColorMode;
```

Comment:
Type this (and `unblended`, `alphaBlended`, `DepthMode.disabled`, etc.) as `$ReadOnly` too.